src/main/java/racingcar/view/RacingCarView.java

```diff
@@ -0,0 +1,58 @@
+package racingcar.view;
+
+import racingcar.service.SystemService;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class RacingCarView{
+    SystemService systemService = new SystemService();
+
+    Scanner scan = new Scanner(System.in);
+    int cycle = 0;
+
+    public RacingCarView(){
+        inputCarInfo();
+        inputCycle();
+        showResult();
+        showWinner();
+    }
+    public void inputCarInfo(){
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        String input = scan.next();
+        systemService.splitString(input);
+    }
+    public void inputCycle() {
+        System.out.println("시도할 횟수는 몇 회인가요?");
+        cycle = scan.nextInt();
+    }
+    public void showResult(){
+        System.out.println("실행결과");
+        for(int i=0;i<cycle;i++){
+            cycleCarInformation();
+        }
+    }
+    public void cycleCarInformation(){
+        for(int i = 0; i< systemService.carInfo.size(); i++){
+            int movedLocation = systemService.MoveCar(systemService.carInfo.get(i).getLocationInfo());
+            systemService.carInfo.get(i).setLocationInfo(movedLocation);
```

Comment:
```suggestion
            systemService.getCarInfo().get(i).setLocationInfo(movedLocation);
```
물론 SystemService가 getter 메서드를 가지고 있지 않기 때문에, getter 메서드를 만들지 않으면 컴파일 오류가 날겁니다 😅
하지만 이야기하고자 하는 것은 객체의 상태값을   ``systemService.calInfo`` 와 같이 직접 가져와서 쓰는 것은 좋지 않습니다. 가급적이면 객체의 상태 값은 ``private `` 접근 제한자로 지정하여서 외부에서 함부로 가져다 쓰지 못하도록 만들어야합니다.